.github/workflows/daily_metrics_rebuild.yml

```diff
@@ -0,0 +1,14 @@
+name: nightly-netlify-build
+
+on:
+  schedule:
+    - cron: "15 9 * * *"
+
```

Comment:
@rwfeather what do you think about adding `workflow_dispatch` so its possible to [manually trigger](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow) this workflow? There might be cases where the upstream fails so it would be helpful to be able to manually rerun this if needed.

Reply:
Good idea, I've updated the action to include it